validator/api/src/test/java/tech/pegasys/teku/validator/api/KeyStoreFilesLocatorTest.java

```diff
@@ -103,6 +104,27 @@ public void shouldIgnoreSomeFiles(@TempDir final Path tempDir) throws IOExceptio
                 tempDir, Path.of("key", "a.json").toString(), Path.of("pass", "a.txt").toString()));
   }
 
+  @Test
+  public void shouldIgnoreDepositDataJsonFile(@TempDir final Path tempDir) throws IOException {
+    createFolders(tempDir, "key", "pass");
+    createFiles(
+        tempDir,
+        Path.of("key", "deposit-data.json"),
```

Comment:
Probably worth making this a bit more realistic and name the file `deposit_data-1620858087.json` since that's the type of name the deposit cli actually generates.